client/src/app.js

```diff
@@ -1592,3 +1594,42 @@ function initgenome(g) {
 	}
 	return null
 }
+
+async function launchDisco(arg, app) {
+	console.log(arg, app)
+	if (!arg.genome) throw '"genome" parameter missing'
+	const genomeObj = app.genomes[arg.genome]
+	if (!genomeObj) throw 'unknown genome'
+	if (!Array.isArray(arg.disco.mlst)) throw 'arg.disco.mlst[] not array'
+	/*
+	const opts = {
+		holder: app.holder0,
+		state: {
+			genome: genomeObj.name,
+			args: {
+				sampleName: 'test',
+				data: arg.disco.mlst,
+				genome: genomeObj
+			},
+
+			plots: [
+				{
+					chartType: 'Disco',
+					subfolder: 'disco',
+					extension: 'ts',
+					overrides: {
+						label: {
+							showPrioritizeGeneLabelsByGeneSets: !!genomeObj.geneset
+						}
+					}
+				}
+			]
+		}
+	}
+	const plot = await import('#plots/plot.app.js')
+	const plotAppApi = await plot.appInit(opts)
+	*/
```

Comment:
above are what's copied from plots/plot.app.js. that requires termdbconfig from an official dataset which does not exist here (data is adhoc, no dslabel). thus i need to be able to directly import disco code and plot
what are the parameters?

Reply:
I think you need the params from `/client/plots/disco/Settings.ts`? Should we rename the script to `disco.settings.ts`?